microservices/services/query-metric/api/src/main/java/datawave/microservice/querymetric/QueryMetric.java

```diff
@@ -111,6 +111,12 @@ public QueryMetric(QueryMetric other) {
                 this.predictions.add(p.duplicate());
             }
         }
+        if (other.subPlans != null) {
+            this.subPlans = new HashMap<>();
+            if (!other.subPlans.isEmpty()) {
+                this.subPlans.putAll(other.subPlans);
```

Comment:
Java String is immutable but Subplans is not.  You'll need to create a new Subplans for each entry in order to properly duplicate the subPlans Map.  A Subplans copy constructor and constructor that takes the two counts will help.